app/assets/javascripts/chart-data-mixin.js

```diff
@@ -0,0 +1,129 @@
+angular.module('containerDashboard').factory('ChartsDataMixin', ['$timeout', '$q', function chartDataMixinFactory ($timeout, $q) {
```

Comment:
> please try to see what is relevant in this file and what's not. currently most of it is not needed

@abonas As I'm trying to integrate the 'Node utilization' cards I'm trying to figure out if ```container_dashboard_service.rb``` should do most of these and give the data ready to be inserted to the card. Your thoughts?